### ISSUE TITLE
Moderator Section 

### ISSUE THREAD

**Issue by DataPirate007** [AAADevs/alpha-bot#21]
### About
The Moderation Commands, these commands would be accessible by Assistant Manager and higher roles.

### Reference Link
https://github.com/cookkkie/mee6 (coded in python)
Dyno Discord Bot

### Commands Needed
![image](https://user-images.githubusercontent.com/66989464/89400094-00805080-d731-11ea-8f86-06608938a655.png)

**Comment by FFlop** [comment]
For the clean command, it just deletes a certain amount of message?

**Comment by FFlop** [comment]
uhh by the way for the warning command if there is more than 3 warnings the person will be kicked.. should i do that?

**Comment by DataPirate007** [comment]
Yup, we also be able to edit, like we can edit the warnings to being kicked out later through a command.

**Comment by FFlop** [comment]
k sounds good i have been making the help commands for specifially the mod cog/module so people can check out what commands the mod module can use
